spec/requests/omniauth_callbacks_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Users::OmniauthCallbacksController, type: :request do
```

Comment:
No need to specify the type if you got automated inferring enabled here -> https://github.com/agoraspeakers/meetings-portal/blob/9cab23752654db2cfbfd0e1e6b102a7776295103/spec/rails_helper.rb#L57

Reply:
Thanks for info! :)